File: 백준/Bronze/2739. 구구단/구구단.cc
#include <stdio.h>

void GuGu(int num) {
	for (int i = 1; i <= 9; i++) {
		printf("%d * %d = %d\n", num, i, num * i);
	}
}

int main(void) {
	int num = 0;

	if (scanf("%d", &num) != 1) return -1;

	GuGu(num);

	return 0;
}

File: 백준/Bronze/2739. 구구단/구구단.py
import sys
input = sys.stdin.readline

def GuGu(num: int):
    for i in range(1, 10):
        print(f"{num} * {i} = {num * i}")

def solve():
    try:
        line = input().rstrip()
        if not line: return
        
        num = int(line)
        GuGu(num)
    except ValueError:
        pass

if __name__ == "__main__":
    solve()
